core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -81,6 +81,15 @@
   List<String> getTabletServers();
 
   /**
+   * List the tablet server status
+   *
+   * @return A list of tablet server status.
+   * @since 2.0.0
+   */
+
+  List<Map<String,String>> getTabletServerStatus() throws AccumuloException;
```

Comment:
Introducing a TabletServerID would be inconsistent with the rest of the API which currently uses a string to identify tablet servers.